docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -236,8 +240,38 @@ GET /v1/resources/{org_label}/{project_label}?from={from}
                                              &q={search}
                                              &sort={sort}
 ```
-                                          
-where...
+
+### Within an organization
+
+```
+GET /v1/resources/{org_label}?from={from}
+                             &size={size}
+                             &deprecated={deprecated}
+                             &rev={rev}
+                             &type={type}
+                             &createdBy={createdBy}
+                             &updatedBy={updatedBy}
+                             &schema={schema}
+                             &q={search}
+                             &sort={sort}
+```
+
+### Within all projects
```

Comment:
It is worth mentioning here that the results will contain only the resources from projects where the caller has access (`resources/read`)